ktor-client/ktor-client-features/ktor-client-json/jvm/src/io/ktor/client/features/json/DefaultJvm.kt

```diff
@@ -17,5 +17,5 @@ actual fun defaultSerializer(): JsonSerializer {
             " - ktor-client-serialization"
     )
 
-    return serializers.maxBy { it::javaClass.name }!!
+    return serializers.maxByOrNull { it::javaClass.name }!!
```

Comment:
This makes it kotlin 1.4.0 only, no compatibility with previously versions. Are we fine with it?

Reply:
Ah, or may be it's ok since it's inline